llvm-15.0.3/llvm-crash-analyzer/lib/Analysis/TaintAnalysis.cpp

```diff
@@ -1009,6 +1009,8 @@ bool crash_analyzer::TaintAnalysis::runOnBlameMF(
             if (MIIt == MBB->rend())
               return Result;
           }
+          // Update PC for CALL instruction.
+          ReverseExecutionRecord.updatePC(*MIIt);
           // Skip processing the call instruction
           ++MIIt;
           if (MIIt == MBB->rend())
```

Comment:
Shouldn't we also call the updatePC (*MIIt) after this if clause to account for the instruction we just incremented?

Reply:
Actually, we are updating PC for that instruction at line 1027 or 1075. Probably, it could happen that both updates are performed for the same instruction, and I will fix that. However, updatePC could be invoked multiple times for the same instruction and it won't harm the analysis. Thanks, for the comment!